Dropdown/index.jsx

```diff
@@ -163,18 +163,24 @@ const Dropdown = React.createClass({
     const hasNonDefaultState = disabled || warning || error
     const useDynamicStyles = customize && !hasNonDefaultState
 
-    const dynamicStyles = useDynamicStyles
+    let dynamicStyles = useDynamicStyles
       ? {
         borderColor: this.state.hover || focus
           ? customize.borderColorSelected
           : customize.borderColor,
-        boxShadow: focus && `0 0 4px ${customize.borderColorSelected}`,
+        boxShadow: focus && `0 0 4px ${customize.borderColorSelected}`
```

Comment:
Let’s do it with a ternary in here instead of a `let`:

```javascript
const dynamicStyles = customize ?
  {
    ...(!hasNonDefaultState && {
      borderColor: this.state.hover || focus
        ? customize.borderColorSelected
        : customize.borderColor,
      boxShadow: focus && `0 0 4px ${customize.borderColorSelected}`
    }),
    ...(customize.borderRadius && stacking.position.getBorderRadii(
      this.props,
      customize.borderRadius
    ))
  }
  : undefined
```